TUM Campus App/CardTableViewCell.swift

```diff
@@ -7,12 +7,41 @@
 //
 
 import UIKit
+import FoldingCell
 import MCSwipeTableViewCell
 
-class CardTableViewCell: MCSwipeTableViewCell {
+class CardTableViewCell: FoldingCell {
+    
     
     func setElement(_ element: DataElement) {
```

Comment:
Also I can no longer use MCSwipeTableViewCell for the cards because they already inherit from FoldingCell